lua/rustowl/init.lua

```diff
@@ -1,35 +1,31 @@
 local M = {}
 
----@class RustOwlOptions
----@field auto_enable? boolean Enable RustOwl immediately
----@field idle_time? number Time in milliseconds to hover with the cursor before triggering RustOwl
----@field client? vim.lsp.ClientConfig | {} LSP client configuration that gets passed to `require('lspconfig').rustowl.setup()`
-local options = {
-  auto_enable = true,
-  idle_time = 500,
-  client = {},
-}
-
----@return RustOwlOptions
-function M.get_options()
-  return options
+--- Enable RustOwl highlighting
+---@param bufnr? number
```

Comment:
luals complains about listing `bufnr` as a parameter that doesn't exist. Perhaps we could just  have `bufnr` as a parameter, and not `...`, all that `require('rustowl.show-on-hover').[enable,disable,toggle]()` take in as parameter is `bufnr` anyway